src/hooks/useLastSafe.ts

```diff
@@ -0,0 +1,13 @@
+import useChainId from '@/hooks/useChainId'
+import { useAppSelector } from '@/store'
+import { selectLastSafeAddress } from '@/store/sessionSlice'
+import chains from '@/config/chains'
+
+const useLastSafe = (): string | undefined => {
+  const chainId = useChainId()
+  const lastSafeAddress = useAppSelector((state) => selectLastSafeAddress(state, chainId))
+  const prefix = Object.keys(chains).find((prefix) => chains[prefix] === chainId)
+  return prefix && lastSafeAddress ? `${prefix}:${lastSafeAddress}` : undefined
+}
+
+export { useLastSafe }
```

Comment:
Why not default?

Reply:
I'm not a big fan of default exports for reasons listed [here](https://github.com/basarat/typescript-book/blob/master/docs/tips/defaultIsBad.md)

Reply:
Most of our hooks and other modules use a default export, so I'd prefer if you follow that pattern.